src/components/quiz.tsx

```diff
@@ -1,102 +1,139 @@
-            <legend className="sr-only">Server size</legend>
-            <div className="space-y-4">
-              {quiz.answers.map((option) => (
-                <Option label={option.option} key={option.value}>
-                  <Field
-                    className="bg-green-600"
-                    type="radio"
-                    name={name}
-                    value={option.value}
-                    checked={values[name] === option.value}
-                  />
-                </Option>
-              ))}
-            </div>
-            <Button className="col-start-2 col-span-4 md:col-start-3 mt-10">
-              Next Question
-            </Button>
-          </Form>
-        )}
-      </Formik>
+        <legend className="sr-only">Server size</legend>
+        <div className="space-y-4">
+          {currentQuestion.options.map((option) => (
+            <Option
+              label={option.value}
+              key={option.value}
+              onClick={() => {
+                setSelectedResponse(option.score)
+              }}
+            >
+              <input
+                className="bg-green-600"
+                tabIndex={-1}
+                type="radio"
+                name={option.value}
+                value={option.score}
+                checked={selectedResponse === option.score}
+                readOnly
+              />
+            </Option>
+          ))}
+        </div>
+        <Button
+          className="col-start-2 col-span-4 md:col-start-3 mt-10"
+          disabled={selectedResponse === defaultSelectedResponse}
+        >
+          Next Question
+        </Button>
+      </form>
     </div>
   )
 }
 
-let Option = ({children, label}: any) => {
+let Option = ({children, label, onClick = () => {}}: any) => {
   const checked = children.props.checked
   const greenBorder = checked
     ? `border-green-500 hover:border-green-600`
     : 'border-gray-30 hover:border-green-600'
   return (
     <label
       className={`relative rounded-lg border-2 0 bg-white shadow-sm px-6 py-4 cursor-pointer  sm:flex sm:justify-between focus-within:ring-1 focus-within:ring-offset-2 focus-within:ring-offset-green-400 flex items-center justify-between ${greenBorder}`}
+      onClick={onClick}
     >
-      {children}
       <div className="flex items-center">
         <div className="text-3xl">
           <p id="server-size-0-label" className="font-light text-gray-900">
             {label}
           </p>
         </div>
       </div>
+      {children}
     </label>
   )
 }
 
 let Button: FunctionComponent<{
   className?: string
-}> = ({children, className}) => {
+  disabled: boolean
+}> = ({children, className, disabled}) => {
   return (
     <button
-      type="button"
-      className={`p-6 border border-transparent text-3xl text-center font-medium rounded-md shadow-sm text-white bg-red-700 hover:bg-red-800 focus:outline-none focus:ring-2 focus:ring-offset-2 focus:ring-red-500 ${
+      type="submit"
```

Comment:
this button needs to by `type=submit` so that it will cause the form to submit when clicked.